src/app.js

```diff
@@ -1,3 +1,28 @@
-const app = async () => '#BuildforSDG';
+const express = require('express');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const cors = require('cors')
 
-export default app;
+const app = express();
+
+// using body parser.
+app.use(bodyParser.urlencoded({ extended: true }));
+
+//enable CORS
+app.use(cors());
+
+// using cookie-parser.
+app.use(cookieParser());
+
+// custom imports.
+const router = require('./routes/router');
+require('dotenv-flow').config();
+
+// calls routes/router middleware.
+app.use(router);
+// server listening.
+app.listen(process.env.PORT, () => {
```

Comment:
# Nitpick 
You should extract `process.env.PORT` into a variable like `port` at the top. This makes the code more readable.